verifiers/proofofsql/Cargo.toml

```
@@ -0,0 +1,53 @@
+[package]
+name = "pallet-proofofsql-verifier"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+homepage.workspace = true
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+proof-of-sql-verifier = { git = "https://github.com/HorizenLabs/proof-of-sql-verifier.git", default-features = false, rev = "2beb019030e104b2a0519a785d2a043db06435a6" }
+log = "0.4.21"
+hex-literal = { workspace = true, optional = true }
+codec = { workspace = true }
+scale-info = { workspace = true }
+derivative = { version = "2.2.0" }
```

Please, switch to `educe` crate: is maintained and well documented.

```suggestion
educe = { version = "0.6.0", default-features = false, features = [
    "Clone",
    "Debug",
    "PartialEq",
] }
```